oss-internship-2020/libarchive/ld_preload_example/sapi_minitar.cc

```diff
@@ -0,0 +1,207 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "sapi_minitar.h"  // NOLINT(build/include)
+
+#include <dlfcn.h>
+
+#include <iostream>
+#include <memory>
+
+#include "libarchive_sapi.sapi.h"  // NOLINT(build/include)
+#include "sandbox.h"               // NOLINT(build/include)
+#include "sandboxed_api/var_array.h"
+#include "sandboxed_api/var_ptr.h"
+
+std::unique_ptr<SapiLibarchiveSandboxExtract> sandbox_extract;
+std::unique_ptr<LibarchiveApi> api;
```

Comment:
See my comment below, these should be regular pointers.

https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables